picard/tagger.py

```diff
@@ -421,6 +428,16 @@ def handle_command_quit(self, argstring):
         self.exit()
         self.quit()
 
+    def handle_command_remove(self, argstring):
+        for file in self.get_all_file_objects():
+            if argstring == file.filename:
+                argstring.remove()
```

Comment:
```suggestion
                file.remove()
```

?

Reply:
Yeah, I was experimenting a bit and finally left with some typo, thanks for noticing